platform/darwin/docs/theme/templates/nav.mustache

```diff
@@ -2,7 +2,7 @@
   <ul class="nav-groups">
     {{#structure}}
     <li class="nav-group-name">
-      <a class="nav-group-name-link" href="{{path_to_root}}{{section}}.html">{{section}}</a>
+      <a class="small-heading" href="{{path_to_root}}{{section}}.html">{{section}}</a>
       <ul class="nav-group-tasks">
         {{#children}}
         <li class="nav-group-task">
```

Comment:
Lots of function names overflow no matter how wide we make the table of contents, and this issue worsens as the window narrows:

<img src="https://cloud.githubusercontent.com/assets/1231218/21218425/64eaf21c-c265-11e6-8940-c8a41834c4d0.png" width="150" alt="toc">

We can mitigate this issue by attaching `title` attributes to each of the links.